lib/cli-engine.js

```diff
@@ -553,6 +554,9 @@ CLIEngine.prototype = {
                     if (warnIgnored) {
                         results.push(createIgnoreResult(filename));
                     }
+                    if (options.verbose) {
+                        log.info("Ignoring: " + filename);
```

Comment:
A bit worried about this. If the output put on stdout? If so, it could disrupt the output format. I think it should end up on stderr to avoid that.